examples/todomvc-footer/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@feature-hub/todomvc-footer",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
+  "main": "dist/index.js",
+  "scripts": {
+    "prebundle": "rimraf dist",
+    "bundle": "parcel build src/index.tsx"
+  },
+  "dependencies": {
+    "@feature-hub/core": "^0.2.0",
+    "@feature-hub/react": "^0.2.0",
+    "@feature-hub/todomvc-todo-manager": "^0.0.0"
+  },
+  "devDependencies": {
+    "@types/react": "^16.7.8",
+    "caniuse-lite": "^1.0.30000912",
+    "parcel-bundler": "^1.10.3",
+    "postcss-cssnext": "^3.1.0",
+    "postcss-modules": "^1.4.1",
+    "react": "^16.6.3",
```

Comment:
why is react a dev dep?

Reply:
It was originally because `react` was declared as external dependency in the webpack config. We should probably do the same with rollup.